app/boards/shields/bfo9000/bfo9000.yaml

```
@@ -0,0 +1,7 @@
+identifier: bfo9000
+name: BFO-9000
+type: shield
+manufacturer: keeb.io
+compatible: Pro Micro
+split: true
```

Instead of a simple split boolean, to support future combinations, e.g. multiple peripherals for a split w/ a numpad that's separate, thoughts on a `components` or somesuch array, e.g.:

```suggestion
components: [left, right]
```

More work for every split board, but makes it easier to unambiguously setup which shields to build for if we automate setting up the template repos off this.

---

Yeah, this is a good idea. Will probably need to be fleshed out more once dongles are a thing (building with/without a dongle in the mix). Maybe even an array of arrays that contain possible set ups?

---

Or "nodes"?

An array is a good start.  We may need more flexible topologies in the future but we should probably figure out the firmware approach to that first.